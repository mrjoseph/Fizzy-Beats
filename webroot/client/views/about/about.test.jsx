import React from 'react';
import renderer from 'react-test-renderer';
import About from './about';

const props = {
  location: {
    search: {
      foo: 'bar',
    },
  },
};

describe('my first test', () => {
  it('should console.log component', () => {
    const tree = renderer
      .create(<About {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
