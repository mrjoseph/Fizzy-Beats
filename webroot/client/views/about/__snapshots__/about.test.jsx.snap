// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my first test should console.log component 1`] = `
<div>
  <h1>
    About my app
     
  </h1>
</div>
`;
